beta-src/src/components/WDPopover.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react";
+import { useRef } from "react";
+import { Box, Popover } from "@mui/material";
+
+interface WDPopoverProps {
+  children: React.ReactNode;
+  isOpen: boolean;
+  /**
+   * A callback to be used when the popover closes
+   */
+  onClose: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
This needs to be optional, you shouldn't be forced to pass a callback when the popover closes.

Reply:
So we decided to control modal functionality from parent so we need to pass that handler to make isOpen false

Reply:
@bryancodazen this onClose is required by popover...we are just passing it through and allowing the parent of this component to control that close behavior.

Reply:
@sariacodazen @nebevets let's sync up on this tomorrow (02/08) during the 15 min standup time if possible.